frontend/src/elements/e-countdown.ts

```diff
@@ -27,19 +27,50 @@ export class ECountdown extends LitElement {
 	protected updated(_changedProperties: Map<string | number | symbol, unknown>): void {
 	  if (_changedProperties.has('timer')) {
 			this.complete = this.timer === 0;
+
+			if (this.complete) {
+				const event: Event = new Event("timerComplete", {
+					bubbles: true,
+					composed: true
+				});
+				this.dispatchEvent(event);
+			}
 		}
 	}
 
 	protected firstUpdated(): void {
-		this._countdown = setInterval(this.updateTimer.bind(this), 1000);
+		this.timer = this.totalSeconds;
```

Comment:
Looks like a good use for a controller

Reply:
I was looking into creating a controller but we won't have any other timing mechanisms in the app so maybe it's overkill? Possibly?

Reply:
Yeah could be. Can always go back if the abstraction becomes useful